plugins/AITagger/ai_tagger.yml

```diff
@@ -0,0 +1,17 @@
+name: AI Tagger
+description: Tag videos and Images with Locally hosted AI using Skier's Patreon AI models
+version: 1.0
+url: https://github.com/stashapp/CommunityScripts/tree/main/plugins/AITagger
+exec:
+  - python
+  - plugins/AITagger/ai_tagger.py
```

Comment:
if you reference the file instead of the base dir, you can avoid the chdir

Reply:
I'm not exactly sure what you mean. Can you give an example?

Reply:
https://github.com/stashapp/CommunityScripts/blob/42c7da20754477c5e7b5abff84e502ab707b1c06/plugins/phashDuplicateTagger/phashDuplicateTagger.yml#L5-L7

Reply:
I tested that approach and with it and removing chdir it broke accessing the local path to the csv. Or are you saying that I can also use that {pluginDir} in the python file as well instead of a relative path?